src/redux/reducer/appointmentSlice.js

```diff
@@ -0,0 +1,112 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const urlappoint = 'https://doctalk-r977.onrender.com/api/v1/appointments';
```

Comment:
[OPTIONaL] Consider using a base url, since endpoint would be the same regardless of base, it would make it easier when testing locally etc.